### PR TITLE
ansi 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by drewsonne**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source ansi`, where `ansi` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install ansi`)?

-----

### CODE REVIEW

File: Formula/ansi.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Ansi < Formula
+  desc "This bash script is designed to help you colorize words and bits of text"
+  homepage "https://github.com/fidian/ansi"
+  url "https://github.com/fidian/ansi.git",
+    :using => :git,
```
Comment (commitay) [inline Formula/ansi.rb]: This line can be removed.

File: Formula/ansi.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    :revision => "3624905213bd89f1023c1190df25069acc14abd5"
```
Comment (commitay) [inline Formula/ansi.rb]: We require a tagged release, not just a commit. Also, tarballs are preferred over git checkouts unless they are needed for a reason, e.g. submodules.

As the most recent tag is almost two years old I suggest that you ask upstream to tag a new release.

File: Formula/ansi.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  version "1.0.0"
+  sha256 "63ab048d3949c490e9c3a3f0f51e1d50a4bd942015289a521bc495ac1eea252f"
+
+  def install
+    bin.install "ansi"
+  end
+
+  test do
+    system "#{bin}/ansi", "--help"
```
Comment (commitay) [inline Formula/ansi.rb]: The test needs to exercise and verify the functionality of the software.

**Review by commitay — CHANGES_REQUESTED** [review]

File: Formula/ansi.rb (hunk 3 above)
Comment (drewsonne) [reply Formula/ansi.rb]: I'm working on this atm. The utility outputs ansi escaped strings, and I'm having difficulty capturing the escape codes from stdout to do a compare.

**Pull request closed without merging**